### PR TITLE
EM-2040

### PR DESCRIPTION
**Pull request by oliv3r**
Can be merged into the XL branch

### CODE REVIEW

File: dts/202970-touchscreen_workstation.dts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ *     HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+ *     WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ *     FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ *     OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include "sun7i-a20-olinuxino-lime2-emmc.dts"
+#include "part201258-ulticontroller3.dtsi"
+
+/ {
```
Comment (MarcoTvM) [inline dts/202970-touchscreen_workstation.dts]: a / ???

File: dts/2621-UM3XL.dts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ *     WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ *     FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ *     OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include "part2621-ultimainboard3.1.dtsi"
+#include "part201258-ulticontroller3.dtsi"
+
+/ {
+	model = "Ultimaker UM3-XL (2621)";
```
Comment (MarcoTvM) [inline dts/2621-UM3XL.dts]: Not XL but S5

File: dts/article.links [hunk 3]
```diff
@@ -2,7 +2,9 @@
 #These articles links are converted to hex representation sym links to be used in the u-boot script to load the proper device tree for this article.
 1797:part2005-withpart1994:programming station
 9066:part2005-withpart1994:UM3
+9066-9051:UM3m:UM3 Hybrid
 9511:part2005-withpart1994:UM3 extended
-9051:part1000897-withpart1994:UM3 XL
+9051:2621-UM3XL:UM3 XL
```
Comment (MarcoTvM) [inline dts/article.links]: XL -> S5

File: dts/part2621-ultimainboard3.1.dtsi [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+			gpios = <&pio 1 11 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+		};
+	};
+};
+
+&i2c1 {
+	cabin_light: pca9632@61 {
+		compatible = "nxp,pca9632";
+		#address-cells = <1>;
```
Comment (MarcoTvM) [inline dts/part2621-ultimainboard3.1.dtsi]: Commented out code should not be checked in

File: dts/part2621-ultimainboard3.1.dtsi [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+		#size-cells = <0>;
```
Comment (MarcoTvM) [inline dts/part2621-ultimainboard3.1.dtsi]: Commented out code should not be checked in

File: dts/202970-touchscreen_workstation.dts [hunk 6]
```diff
@@ -0,0 +1,51 @@
+/*
```
Comment (oliv3r) [inline dts/202970-touchscreen_workstation.dts]: Hello world!

Comment (oliv3r) [reply dts/202970-touchscreen_workstation.dts]: works fine

**Pull request merged**